easybuild/easyblocks/t/trinity.py

```diff
@@ -57,6 +57,20 @@ def __init__(self, *args, **kwargs):
 
         self.build_in_installdir = True
 
+        version = LooseVersion(self.version)
+        if version >= LooseVersion('2.0') and version < LooseVersion('2.3'):
+            sep = '-'
+        elif version >= LooseVersion('2.3') and version < LooseVersion('2.9'):
+            sep = '-Trinity-v'
+        elif version >= LooseVersion('2.9') and version < LooseVersion('3.0'):
+            sep = '-v'
+        else:
+            sep = '_r'
+        self.trinityrnaseq_subdir = 'trinityrnaseq%s%s' % (sep, self.version)
```

Comment:
:heart: f-strings :heart:
```suggestion
        self.trinityrnaseq_subdir = f'trinityrnaseq{sep}{self.version}'
```